### PR TITLE
Update WRITING-STYLE.md

### PR DESCRIPTION
**Pull request by jackquix**
Added recommendation on formatting of text substitution prompt.

## Description

A PR description indicating the purpose of the PR.

* [GitHub issue](link) - link to GitHub issue (if there is one)

## Review

Instructions on how to review the PR. 

* [Page to review](link)

### CODE REVIEW

File: WRITING-STYLE.md [hunk 1]
```diff
@@ -344,6 +344,8 @@ When inserting example code in the text:
 * There should *not* be a space before the colon.
 * Place a blank line after the colon and before the code block.
 
+When substitutions are required in code blocks, for example in requiring the reader to write their own API key, write the text prompt enclosed in a less than sign and a greater than sign.
```
Comment (tbedford) [inline WRITING-STYLE.md]: I would put this in a new section at this point, called "replaceable text" or something similar. The reason is we sometimes have inline code with replaceable text - not just in codeblocks.

I think adding a simple example (or two) would be useful too.

**Review by tbedford — CHANGES_REQUESTED** [review]

File: WRITING-STYLE.md (hunk 1 above)
Comment (jackquix) [reply WRITING-STYLE.md]: Yes, good point! The page is quite example-light so I didn't know, but I will add one

File: WRITING-STYLE.md [hunk 2]
```diff
@@ -344,7 +344,11 @@ When inserting example code in the text:
 * There should *not* be a space before the colon.
 * Place a blank line after the colon and before the code block.
 
-When substitutions are required in code blocks, for example in requiring the reader to write their own API key, write the text prompt enclosed in a less than sign and a greater than sign. 
+## Substitution Prompts
```
Comment (tbedford) [inline WRITING-STYLE.md]: Use sentence case, that is `Substitution prompts`.

File: WRITING-STYLE.md [hunk 3]
```diff
@@ -344,7 +344,11 @@ When inserting example code in the text:
...
+
+When substitutions are required in code blocks, such as requiring the reader to write their own API key, write the text prompt enclosed in a less than sign and a greater than sign. For example: `<your_API_key>`
```
Comment (tbedford) [inline WRITING-STYLE.md]: End sentence with period.

**Review by tbedford — CHANGES_REQUESTED** [review]

**Review by tbedford — APPROVED** [review]

**Pull request merged**